restAPI_ecommerce/src/main/java/e_commerce/restAPI_ecommerce/RestApiEcommerceApplication.java

```diff
@@ -0,0 +1,13 @@
+package e_commerce.restAPI_ecommerce;
```

Comment:
Don't add '_' into package/class names, and use only lowercase
```suggestion
package ecommerce.restapiecommerce;
```